include/internal/list.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 1995-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2002, Oracle and/or its affiliates. All rights reserved
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef HEADER_LIST_H
+# define HEADER_LIST_H
+
+# include <stddef.h>
+# include <openssl/e_os2.h>
+
+# define container_of(ptr, type, member)                \
+    ((type *)((char *)ptr - offsetof(type, member)))
+
+# define list_for_each(iter, head)                                  \
+    for (iter = (head)->next; iter != (head); iter = iter->next)
+
+# define list_for_each_safe(iter, tmp, head)                            \
+    for (iter = (head)->next, tmp = iter->next; iter != (head); iter = tmp, tmp = iter->next)
+
+# define list_for_each_r(iter, head) \
+    for (iter = (head)->prev; iter != (head); iter = iter->prev)
+
+struct list {
+    struct list *next, *prev;
+};
+
+static ossl_inline int list_empty(const struct list *head)
+{
+    return head->next == head;
+}
+
+static ossl_inline size_t list_size(const struct list *head)
+{
+    size_t sz = 0;
+    struct list *iter;
+    for (iter = (head)->next; iter != (head); iter = iter->next)
```

Comment:
blank line after all declarations.  not commenting on that to avoid noise.

Reply:
Resolved: https://github.com/ckalina/openssl/commit/48e5e201d501933af7f21760ced22deec4fc29f3

Reply:
This nit is still present in this file.
Code rebase issue ?